pkg/discoveredpolicy/config/prometheus/monitor.yaml

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
Hey, what's this prometheus monitor service?

Reply:
this is part of kubeuilder project boilerplate to setup Prometheus use-cases, currently it's not being in used.

Reply:
Can you please remove all the non-required part?